plasmapy/tests/test_requirements.py

```diff
@@ -0,0 +1,141 @@
+"""Tests for the consistency of requirements."""
+
+import os
+import pytest
+import setuptools
+import tomli
+
+from typing import Dict, Set
+
+import plasmapy
+
+base_directory = os.path.realpath(f"{plasmapy.__path__[0]}/..")
+requirements_directory = f"{base_directory}/requirements"
```

Comment:
Just use pathlib, like this:

```suggestion
import pathlib
base_directory = pathlib.Path(__file__).parent.parent
requirements_directory = base_directory / "requirements"
```

this will then also work on Windows. I used `__file__` to dodge weirdness with pip installed versions.